app/code/core/Mage/Adminhtml/Block/System/Convert/Gui/Edit/Tab/Wizard.php

```diff
@@ -92,7 +92,7 @@ public function getValue($key, $default = '', $defaultNew = null)
         }
 
         $value = $this->getData($key);
-        return $this->escapeHtml(strlen($value) > 0 ? $value : $default);
+        return $this->escapeHtml(is_string($value) && strlen($value) > 0 ? $value : $default);
```

Comment:
Preserve original behavior to throw a fatal error if `$value` is a wrong type.
```suggestion
        return $this->escapeHtml($value !== null && strlen($value) > 0 ? $value : $default);
```